src/passes/OptimizeInstructions.cpp

```diff
@@ -459,6 +459,46 @@ struct OptimizeInstructions
         }
         // note that both left and right may be consts, but then we let
         // precompute compute the constant result
+      } else if (binary->op == LtSInt32) {
+        if (auto* c = binary->right->dynCast<Const>()) {
+          auto constValue = c->value.geti32();
+          if (constValue == 0) {
+            // this may increase size for compressed binaryes so it apply
+            // only for non-zero shrinkLevel levels
+            auto options = getPassOptions();
+            if (options.optimizeLevel >= 2 && options.shrinkLevel <= 1) {
```

Comment:
@kripken Is it enough restriction? Or better use `options.shrinkLevel != 0`?

Reply:
I think this can be just `if (options.shrinkLevel == 0)`